### PR TITLE
Support for `pointer_to_address` instruction [ISSUE# 81]

### LINKED ISSUE #81: Support `pointer_to_address` instruction

### PR DESCRIPTION
**Pull request by taufique71**
Changes to handle `pointer_to_address` instruction

Resolves [Issue 81](https://github.com/themaplelab/swift/issues/81).

### CODE REVIEW

File: lib/WALASupport/SILWalaInstructionVisitor.cpp [hunk 1]
```diff
@@ -1421,6 +1421,23 @@ jobject SILWalaInstructionVisitor::visitAddressToPointerInst(AddressToPointerIns
   return RawPointerNode;
 }
 
+jobject SILWalaInstructionVisitor::visitPointerToAddressInst(PointerToAddressInst *PTAI) {
+
```
Comment (karimhamdanali) [inline lib/WALASupport/SILWalaInstructionVisitor.cpp]: remove this empty line

**Review by karimhamdanali — CHANGES_REQUESTED** [review]

**Comment by mmroz** [comment]
So, I looked at the two instructions address_to_pointer and pointer_to_address and these 2 instructions are conversion instructions even though the types are internal. In the PR https://github.com/themaplelab/swift/pull/138 I made that change address_to_pointer and I think we should do that here too! Just add:

```
jobject ConvertedTypeNode = Wala->makeConstant(CovertedType.c_str());
jobject RawPointerCastNode = Wala->makeNode(CAstWrapper::CAST, ConvertedNode, ConvertedTypeNode);
```

**Comment by karimhamdanali** [comment]
In addition to @mmroz comments, please resolve the merge conflicts before I approving this PR.

**Comment by taufique71** [comment]
@karimhamdanali @mmroz, everything resolved. Please take a look if it's okay now.

**Comment by karimhamdanali** [comment]
Thanks @taufique71. @mmroz, do these changes address your concerns as well?

**Comment by mmroz** [comment]
Looks good!

**Review by karimhamdanali — APPROVED** [review]

**Pull request merged**